### PR TITLE
#101 upgrade to log4j2

### PR DESCRIPTION
**Pull request by valliman**
A fresh PR for the issue #101

### CODE REVIEW

**Comment by valliman** [comment]
@jaimecasero one month is gone, can I someone help you to speed things up? Any question from your side concering this one?

**Comment by jaimecasero** [comment]
@valliman  i made some comments on latest PR, still unanswered...

File: src/test/tck/msgflow/callflows/refer/Referrer.java [hunk 1]
```diff
@@ -72,16 +70,6 @@
 
     private static Logger logger = Logger.getLogger(Referrer.class);
 
-    static {
-        try {
-            logger.setLevel(Level.INFO);
-            logger.addAppender(new FileAppender(new SimpleLayout(),
```
Comment (valliman) [inline src/test/tck/msgflow/callflows/refer/Referrer.java]: The provided config files cover this.

**Comment by jaimecasero** [comment]
@valliman i realized log4k2 2.8 requires java 1.7 . Currently source code in jainSIPRi is set to 1.5, so we have an issue here. Let me figure out what to do next...

**Comment by valliman** [comment]
Yay ... Java 1.5. I understand, that's bad.

**Pull request merged**